src/plugins/intel_npu/src/plugin/npuw/partitioning/patterns/opt.cpp

```diff
@@ -1536,10 +1537,10 @@ ConvToMatmul::ConvToMatmul(Context::Ref ctx) {
             auto new_reshape2 = std::make_shared<ov::op::v1::Reshape>(matched_node_param2, new_const2, false);
 
             // Connect to Reshape
-            if (ov::op::util::is_parameter(matched_node_param2)) {
+            if (matched_node_param2_out == matched_node_multiply) {
```

Comment:
...then the logic here becomes clear

```suggestion
            if (cvt2_or_multiply == matched_node_multiply) {
```

Reply:
Done